lib/Sema/CSApply.cpp

```diff
@@ -8217,6 +8217,17 @@ Expr *ExprRewriter::finishApply(ApplyExpr *apply, Type openedType,
         // Resolve into a MakeTemporarilyEscapableExpr.
         auto *args = apply->getArgs();
         assert(args->size() == 2 && "should have two arguments");
+
+        auto appliedWrappers =
```

Comment:
It might be simpler to call coerceToRValue() instead of coerceCallArguments() here. Eg, there aren't any property wrappers on `withoutActuallyEscaping` parameters